packages/node-opentelemetry/src/otel.ts

```diff
@@ -74,25 +115,7 @@ export const initSDK = (config: SDKConfig) => {
           }),
         }),
     instrumentations: [
-      getNodeAutoInstrumentations({
-        '@opentelemetry/instrumentation-http': config.advancedNetworkCapture
-          ? getHyperDXHTTPInstrumentationConfig({
-              httpCaptureHeadersClientRequest:
-                env.OTEL_INSTRUMENTATION_HTTP_CAPTURE_HEADERS_CLIENT_REQUEST,
-              httpCaptureHeadersClientResponse:
-                env.OTEL_INSTRUMENTATION_HTTP_CAPTURE_HEADERS_CLIENT_RESPONSE,
-              httpCaptureHeadersServerRequest:
-                env.OTEL_INSTRUMENTATION_HTTP_CAPTURE_HEADERS_SERVER_REQUEST,
-              httpCaptureHeadersServerResponse:
-                env.OTEL_INSTRUMENTATION_HTTP_CAPTURE_HEADERS_SERVER_RESPONSE,
-            })
-          : { enabled: true },
-        // FIXME: issue detected with fs instrumentation (infinite loop)
-        '@opentelemetry/instrumentation-fs': {
-          enabled: false,
-        },
-        ...config.instrumentations,
-      }),
+      getNodeAutoInstrumentations(nodeAutoInstrumentationInputConfig),
```

Comment:
we don't want to bypass all configs to the `getNodeAutoInstrumentations` since it only support a specific set of pkgs (https://github.com/open-telemetry/opentelemetry-js-contrib/blob/7ef45fa742a3d79d607810ed5f5a212f6db56509/metapackages/auto-instrumentations-node/src/utils.ts#L86)

Reply:
Gotcha, so putting it outside the getNodeAutoInstrumentations, but inside the instrumentations array gives me a type error with ...config.additionalInstrumentations, it wants me to use the type InstrumentationBase instead of InstrumentationOption. I assume that is still fine?

Reply:
Yeah, I think the type is `Instrumentation`

Reply:
Gotcha perfect! Trying to test it now with Yarn link.